src/main/java/io/xdb/core/client/Client.java

```diff
@@ -129,7 +128,11 @@ private String startProcessInternal(final String processType, final String proce
             : process.getOptions();
 
         final ProcessExecutionStartRequest request = new ProcessExecutionStartRequest()
-            .namespace(processOptions.getNamespace())
+            .namespace(
+                Strings.isNullOrEmpty(processOptions.getNamespace())
```

Comment:
I feel like it's confusing to have two places defining the namespace. 
WDYT if we remove the one in processOptions?

Reply:
Makes sense. Then maybe we should also remove namespace from stopProcess/describeProcess?

Reply:
Yeah good point. Agreed